### DISCUSSION TITLE
Unique Index null not distinct with postgres

### DISCUSSION THREAD

**Discussion by guedesfelipe** [sqlalchemy/sqlalchemy#9764] [Usage Questions] [answered]
Hi! How I can create that index with SQLAlchemy? (Postgres)

```sql
CREATE UNIQUE INDEX uk_test ON table_test (column1, column2) NULLS NOT DISTINCT;
```

I created it manually inside a migration generated by the still automatically, but when generating new migrations it ends up being deleted. I would like to know if there is a way for me to create this index using SQLAlchemy or if there is any other way to do it.

**Comment by CaselIT** [comment, marked as answer]
Hi,

This is currently not supported. The feature is tracked here https://github.com/sqlalchemy/sqlalchemy/issues/8240. As always PR are welcome

**Reply by guedesfelipe** [reply]
Thank you! I will check how to do it!
